docs/user/learner.rst

```diff
@@ -0,0 +1,130 @@
+Learner
+===============
+
+Accessing Kolibri
+-----------------
+
+To log in to **Kolibri** follow these steps:
+
+#. Click the **Log In** icon in the main menu (left or bottom) to open the login window.
+#. Type the username and password provided by your **Coach** or **Admin**.
```

Comment:
Technically, only Admins have permission to reset passwords. However it does seem reasonable that they might give coaches a user/password list?